duckduckgo/iOS
# remove mac promo experiment

Task/Issue URL: https://app.asana.com/0/0/1204731397004150/f
Tech Design URL:
CC:

BSK: https://github.com/duckduckgo/BrowserServicesKit/pull/401

**Description**:

Removes Mac promo experiment while adding share link/text support to the RMF

**Steps to test this PR**:
1. Clean install, run the app and get through onboarding.
2. Restart the app, a sample message should display showing an RMF message.
3. Check RMF pixels fire as expected
4. Using the share sheet should *NOT* dismiss the RMF



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
